test/calico/metadata_access_test.go

```diff
@@ -89,7 +89,7 @@ func TestNoMetadataAccessRandomPod(t *testing.T) { //nolint:funlen
 	platform := os.Getenv("PLATFORM")
 
 	switch platform {
-	case testutil.PlatformPacket, testutil.PlatformPacketARM:
+	case testutil.PlatformPacket, testutil.PlatformPacketARM, "packet-arm":
```

Comment:
We can add a TODO to remove this once the CI config is updated. But if you're doing it right after this is merged don't bother with the TODO.

Reply:
Yeah, will do it right after :)